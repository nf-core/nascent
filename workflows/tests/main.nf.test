nextflow_pipeline {

    name "Test Workflow main.nf"
    script "../../main.nf"

    test("Should run with bwa (default)") {

        when {
            params {
                outdir = "$outputDir"
                aligner = "bwa"
            }
        }

        then {
            assert workflow.success
            assert snapshot(
                workflow.trace.tasks().size(),
                path("$outputDir/bbmap").list(),
                // BUG https://github.com/nf-core/nascent/issues/102
                // path("$outputDir/bedtools").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/bwa/featurecounts/gene").list(),
                // FIXME cd4 isn't deterministic
                // path("$outputDir/bwa/featurecounts/predicted").list(),
                // path("$outputDir/deeptools").list(),
                // path("$outputDir/fastp/*.json").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/gtf2bed").list(),
                path("$outputDir/homer/cd4.bed"),
                path("$outputDir/homer/jurkat.bed"),
                path("$outputDir/preseq").list(),
                // path("$outputDir/rseqc").list(),
                path("$outputDir/samtools").list(),
            ).match()
        }

    }

    test("Should run with bwamem2") {

        when {
            params {
                outdir = "$outputDir"
                aligner = "bwamem2"
            }
        }

        then {
            assert workflow.success
            assert snapshot(
                workflow.trace.tasks().size(),
                path("$outputDir/bbmap").list(),
                // BUG https://github.com/nf-core/nascent/issues/102
                // path("$outputDir/bedtools").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/bwa/featurecounts/gene").list(),
                // FIXME cd4 isn't deterministic
                // path("$outputDir/bwa/featurecounts/predicted").list(),
                // path("$outputDir/deeptools").list(),
                // path("$outputDir/fastp/*.json").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/gtf2bed").list(),
                path("$outputDir/homer/cd4.bed"),
                path("$outputDir/homer/jurkat.bed"),
                path("$outputDir/preseq").list(),
                // path("$outputDir/rseqc").list(),
                path("$outputDir/samtools").list(),
            ).match()
        }

    }

    test("Should run with dragmap") {

        when {
            params {
                outdir = "$outputDir"
                aligner = "dragmap"
            }
        }

        then {
            assert workflow.success
            assert snapshot(
                workflow.trace.tasks().size(),
                path("$outputDir/bbmap").list(),
                // BUG https://github.com/nf-core/nascent/issues/102
                // path("$outputDir/bedtools").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/bwa/featurecounts/gene").list(),
                // FIXME cd4 isn't deterministic
                // path("$outputDir/bwa/featurecounts/predicted").list(),
                // path("$outputDir/deeptools").list(),
                // path("$outputDir/fastp/*.json").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/gtf2bed").list(),
                path("$outputDir/homer/cd4.bed"),
                path("$outputDir/homer/jurkat.bed"),
                path("$outputDir/preseq").list(),
                // path("$outputDir/rseqc").list(),
                // FIXME Not deterministic
                // path("$outputDir/samtools").list(),
            ).match()
        }
    }

    test("Should run with HISAT2") {

        tag "HISAT2"

        when {
            params {
                outdir = "$outputDir"
                aligner = "hisat2"
                hisat2_index = 'https://raw.githubusercontent.com/nf-core/test-datasets/nascent/reference/GRCh38_chr21_hisat2.tar.gz'
            }
        }

        then {
            assert workflow.success
            assert snapshot(
                workflow.trace.tasks().size(),
                path("$outputDir/bbmap").list(),
                // BUG https://github.com/nf-core/nascent/issues/102
                // path("$outputDir/bedtools").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/bwa/featurecounts/gene").list(),
                // FIXME cd4 isn't deterministic
                // path("$outputDir/bwa/featurecounts/predicted").list(),
                // path("$outputDir/deeptools").list(),
                // path("$outputDir/fastp/*.json").list(),
                // FIXME gtf2bed isn't deterministic
                // path("$outputDir/gtf2bed").list(),
                path("$outputDir/homer/cd4.bed"),
                path("$outputDir/homer/jurkat.bed"),
                path("$outputDir/preseq").list(),
                // path("$outputDir/rseqc").list(),
                // FIXME Not deterministic
                // path("$outputDir/samtools").list(),
            ).match()
        }
    }
}
