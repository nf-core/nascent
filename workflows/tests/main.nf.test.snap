{
    "Should run with bwa (default)": {
        "content": [
            128,
            [
                "cd4_REP1.coverage.hist.txt:md5,7cbb473be8d3b32ff2e52fdf4e5d10d2",
                "cd4_REP1.coverage.stats.txt:md5,f1471b61ac17dba283d80e08450c7e55",
                "cd4_REP2.coverage.hist.txt:md5,8d5258a0882494bc4e3f1aa6aa5ed685",
                "cd4_REP2.coverage.stats.txt:md5,3cdb4473211f9da44166ffa6aaa5b602",
                "cd4_REP3.coverage.hist.txt:md5,4a8397c1ea08f35ca7046e3c2c014a08",
                "cd4_REP3.coverage.stats.txt:md5,b88592b469d504a93f2389255ac15f38",
                "cd4_REP4.coverage.hist.txt:md5,d2cbf89e1f753d99bcd03b4e472270dd",
                "cd4_REP4.coverage.stats.txt:md5,4c2b5fe6793aab8560cc87c176ac06ab",
                "jurkat_REP1.coverage.hist.txt:md5,71a893d9d1d55fd47399f6d47b628d6e",
                "jurkat_REP1.coverage.stats.txt:md5,381c69a30099d82066a959deab1a2569",
                "jurkat_REP2.coverage.hist.txt:md5,d124bdadaf2dc5cdd9e3db5a1ab4c588",
                "jurkat_REP2.coverage.stats.txt:md5,b1940346719e4070091f2eef6e508f49"
            ],
            "cd4.bed:md5,ab94918610a560772fdbec591200295f",
            "jurkat.bed:md5,41908723fa423a78476feb2e7627a3a4",
            [
                "cd4_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP1.lc_extrap.txt:md5,6d0e7f782f34ee54213c022cbe19df5b",
                "cd4_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP2.lc_extrap.txt:md5,6aac6dd78dfd85a18fadd20be7377019",
                "cd4_REP3.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP3.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP3.lc_extrap.txt:md5,02c4b0fc0afd282a402a316d44598173",
                "cd4_REP4.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP4.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP4.lc_extrap.txt:md5,9efbab9d2aee791e7e9cae9237f2c500",
                "jurkat_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP1.lc_extrap.txt:md5,390386e428ce783b3a373a5f60ee912f",
                "jurkat_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP2.lc_extrap.txt:md5,cfdda91d83dd381dcefdc754e4b4c54c"
            ],
            [
                "cd4_REP1.flagstat:md5,863e2d506d5cc4239af98a5f31bbc906",
                "cd4_REP1.idxstats:md5,b1dd8bcbd23c53c21f0e11082d9315f2",
                "cd4_REP1.sorted.bam:md5,11a805cfc0bde874d6a7de7467ca25cc",
                "cd4_REP1.sorted.bam.bai:md5,85c4acc92dc079ec42adb5fbea6cbcce",
                "cd4_REP1.stats:md5,1a937e63fb7b1f8063121d8f14f4e918",
                "cd4_REP2.flagstat:md5,0fd86dbf8f799fad49ba471702979bdc",
                "cd4_REP2.idxstats:md5,53204e4c6a9f68664087e4a8123be46a",
                "cd4_REP2.sorted.bam:md5,76bf40b388dc71e35c05792bec0b44c6",
                "cd4_REP2.sorted.bam.bai:md5,2a8ea9af3c3ac5c6923aac765850b833",
                "cd4_REP2.stats:md5,6ac59691b72f0ccbe2dfa4408586a39b",
                "cd4_REP3.flagstat:md5,793c9cadbf80e35dbae3774e9b5bd93f",
                "cd4_REP3.idxstats:md5,42de227ff397013cf288759871b0e938",
                "cd4_REP3.sorted.bam:md5,39bdb1f466ce3fa508547b2b7aa1ca92",
                "cd4_REP3.sorted.bam.bai:md5,f30a9cd423736d71cbe25a9cd685cb54",
                "cd4_REP3.stats:md5,43cfc143cb086a7fa22a0c8d9d1fc9bb",
                "cd4_REP4.flagstat:md5,ed6c609505097f3cc9d9902d69e016f8",
                "cd4_REP4.idxstats:md5,959dc185ae59de4cebe374026711a55e",
                "cd4_REP4.sorted.bam:md5,8dfe41ab8ad05dcca85cafb8797c8db0",
                "cd4_REP4.sorted.bam.bai:md5,2982bfa9b766a29bb8291a56e58855d5",
                "cd4_REP4.stats:md5,e91a8bd8d0918555892235999ff15b1e",
                "jurkat_REP1.flagstat:md5,fd5f02b0f02a407447b804b1d80f5421",
                "jurkat_REP1.idxstats:md5,c61af0847c1ad76c06a8de2815975b32",
                "jurkat_REP1.sorted.bam:md5,ef3c7becc330cff76bf1d3490b743c8b",
                "jurkat_REP1.sorted.bam.bai:md5,9af71e18c414c16dbb28dd3330157d60",
                "jurkat_REP1.stats:md5,895eddd56d3fc3487f54649dcf063fc7",
                "jurkat_REP2.flagstat:md5,86ed47bd41a745ab59de473082c7742d",
                "jurkat_REP2.idxstats:md5,3db8f88c1f836eb5d10f37ca1df81ae9",
                "jurkat_REP2.sorted.bam:md5,7dfe0b9eb0918426bf03f639114d571f",
                "jurkat_REP2.sorted.bam.bai:md5,ad78f94a551665d02177debe97df74db",
                "jurkat_REP2.stats:md5,9466c2094bb843e845d1c15d25aec2ab"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-19T21:59:19.074977598"
    },
    "Should run with bwamem2": {
        "content": [
            128,
            [
                "cd4_REP1.coverage.hist.txt:md5,7cbb473be8d3b32ff2e52fdf4e5d10d2",
                "cd4_REP1.coverage.stats.txt:md5,f1471b61ac17dba283d80e08450c7e55",
                "cd4_REP2.coverage.hist.txt:md5,8d5258a0882494bc4e3f1aa6aa5ed685",
                "cd4_REP2.coverage.stats.txt:md5,3cdb4473211f9da44166ffa6aaa5b602",
                "cd4_REP3.coverage.hist.txt:md5,4a8397c1ea08f35ca7046e3c2c014a08",
                "cd4_REP3.coverage.stats.txt:md5,b88592b469d504a93f2389255ac15f38",
                "cd4_REP4.coverage.hist.txt:md5,d2cbf89e1f753d99bcd03b4e472270dd",
                "cd4_REP4.coverage.stats.txt:md5,4c2b5fe6793aab8560cc87c176ac06ab",
                "jurkat_REP1.coverage.hist.txt:md5,71a893d9d1d55fd47399f6d47b628d6e",
                "jurkat_REP1.coverage.stats.txt:md5,381c69a30099d82066a959deab1a2569",
                "jurkat_REP2.coverage.hist.txt:md5,d124bdadaf2dc5cdd9e3db5a1ab4c588",
                "jurkat_REP2.coverage.stats.txt:md5,b1940346719e4070091f2eef6e508f49"
            ],
            "cd4.bed:md5,ab94918610a560772fdbec591200295f",
            "jurkat.bed:md5,41908723fa423a78476feb2e7627a3a4",
            [
                "cd4_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP1.lc_extrap.txt:md5,6d0e7f782f34ee54213c022cbe19df5b",
                "cd4_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP2.lc_extrap.txt:md5,6aac6dd78dfd85a18fadd20be7377019",
                "cd4_REP3.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP3.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP3.lc_extrap.txt:md5,02c4b0fc0afd282a402a316d44598173",
                "cd4_REP4.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP4.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP4.lc_extrap.txt:md5,9efbab9d2aee791e7e9cae9237f2c500",
                "jurkat_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP1.lc_extrap.txt:md5,390386e428ce783b3a373a5f60ee912f",
                "jurkat_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP2.lc_extrap.txt:md5,cfdda91d83dd381dcefdc754e4b4c54c"
            ],
            [
                "cd4_REP1.flagstat:md5,863e2d506d5cc4239af98a5f31bbc906",
                "cd4_REP1.idxstats:md5,b1dd8bcbd23c53c21f0e11082d9315f2",
                "cd4_REP1.sorted.bam:md5,0ef96c52d334ac02709b65cdd603f869",
                "cd4_REP1.sorted.bam.bai:md5,89c4319f5f28020940ba31ca7e602e84",
                "cd4_REP1.stats:md5,1a937e63fb7b1f8063121d8f14f4e918",
                "cd4_REP2.flagstat:md5,0fd86dbf8f799fad49ba471702979bdc",
                "cd4_REP2.idxstats:md5,53204e4c6a9f68664087e4a8123be46a",
                "cd4_REP2.sorted.bam:md5,298c44c1b77d008aa5a8f3e77dfae0c8",
                "cd4_REP2.sorted.bam.bai:md5,f87fb585bc0d570a16d5ae9230f36113",
                "cd4_REP2.stats:md5,6ac59691b72f0ccbe2dfa4408586a39b",
                "cd4_REP3.flagstat:md5,793c9cadbf80e35dbae3774e9b5bd93f",
                "cd4_REP3.idxstats:md5,42de227ff397013cf288759871b0e938",
                "cd4_REP3.sorted.bam:md5,836cda61a69c692394ae8aebb3af89d7",
                "cd4_REP3.sorted.bam.bai:md5,39c4c10e041e812be47b20996c004594",
                "cd4_REP3.stats:md5,43cfc143cb086a7fa22a0c8d9d1fc9bb",
                "cd4_REP4.flagstat:md5,ed6c609505097f3cc9d9902d69e016f8",
                "cd4_REP4.idxstats:md5,959dc185ae59de4cebe374026711a55e",
                "cd4_REP4.sorted.bam:md5,6778d910ddfabde8e13de2687833e8b7",
                "cd4_REP4.sorted.bam.bai:md5,8cb9c3c74faf7c3594bdbd9fd642434d",
                "cd4_REP4.stats:md5,e91a8bd8d0918555892235999ff15b1e",
                "jurkat_REP1.flagstat:md5,fd5f02b0f02a407447b804b1d80f5421",
                "jurkat_REP1.idxstats:md5,c61af0847c1ad76c06a8de2815975b32",
                "jurkat_REP1.sorted.bam:md5,dd813d4d4f076a1a6317c4f3578f60f5",
                "jurkat_REP1.sorted.bam.bai:md5,78490e10f957ad2b0b05d1106e67d3d5",
                "jurkat_REP1.stats:md5,895eddd56d3fc3487f54649dcf063fc7",
                "jurkat_REP2.flagstat:md5,86ed47bd41a745ab59de473082c7742d",
                "jurkat_REP2.idxstats:md5,3db8f88c1f836eb5d10f37ca1df81ae9",
                "jurkat_REP2.sorted.bam:md5,910b84a2697b85801cf8b2711c4fae5f",
                "jurkat_REP2.sorted.bam.bai:md5,eff0f3690baa030685f76a0687301827",
                "jurkat_REP2.stats:md5,9466c2094bb843e845d1c15d25aec2ab"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-19T22:01:36.548364874"
    },
    "Should run with dragmap": {
        "content": [
            128,
            [
                "cd4_REP1.coverage.hist.txt:md5,bff6862bc8e2015eef927b40671dec1b",
                "cd4_REP1.coverage.stats.txt:md5,c1e912cd0bebc51b289160ff7ab3365a",
                "cd4_REP2.coverage.hist.txt:md5,edadf440d1c3658118ffd6b6a46821f8",
                "cd4_REP2.coverage.stats.txt:md5,cb1ce057d05fc53e7ad9b0b2e3d5aaad",
                "cd4_REP3.coverage.hist.txt:md5,096ce6905c96587ba80ebcbe5893cf3e",
                "cd4_REP3.coverage.stats.txt:md5,f4b88f4eebab0c010a1cfdc3404dfafb",
                "cd4_REP4.coverage.hist.txt:md5,01f93b5c248e0b78f6d94e28eee3f9e7",
                "cd4_REP4.coverage.stats.txt:md5,a4c6f61be8263e7461f8b961cfcc9053",
                "jurkat_REP1.coverage.hist.txt:md5,468c864f655947edb3b306cb7a72633d",
                "jurkat_REP1.coverage.stats.txt:md5,338f3095395c8d8f172e0e27f9a5a60a",
                "jurkat_REP2.coverage.hist.txt:md5,03301957b2f29c43cb8ea3462998c315",
                "jurkat_REP2.coverage.stats.txt:md5,c3fc3f11e84f4d075ba95adbdbb2ee29"
            ],
            "cd4.bed:md5,e9380700e48c1e52f76a2bb3fa4f59d9",
            "jurkat.bed:md5,fe88d4dc3c100262d7aec484e15b1c73",
            [
                "cd4_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP1.lc_extrap.txt:md5,5ecde39eb3bbec0b056b57a5f95f10e7",
                "cd4_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP2.lc_extrap.txt:md5,abb6a284d959110d35dd3b9659244188",
                "cd4_REP3.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP3.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP3.lc_extrap.txt:md5,af11536a40238dc3417468a2f670126c",
                "cd4_REP4.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP4.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP4.lc_extrap.txt:md5,b36b409ec88047cd354eb04a42b852e5",
                "jurkat_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP1.lc_extrap.txt:md5,024057a6211bf22384df658bce250d21",
                "jurkat_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP2.lc_extrap.txt:md5,33bdd91e7df616988c1ec043818b346b"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-05T15:58:59.071139821"
    },
    "Should run with HISAT2": {
        "content": [
            128,
            [
                "cd4_REP1.coverage.hist.txt:md5,dff9783a6054f490bd106a5ea59ff323",
                "cd4_REP1.coverage.stats.txt:md5,bd2ae14e43e59332d62af60601f95368",
                "cd4_REP2.coverage.hist.txt:md5,ca9e862c16d556b3f1baa230d565e0a7",
                "cd4_REP2.coverage.stats.txt:md5,14d8e9d53e129de3caf98fe6c8f52ace",
                "cd4_REP3.coverage.hist.txt:md5,411058935790bff32f9e6d9b7d136597",
                "cd4_REP3.coverage.stats.txt:md5,46bcbba43196873e663d3fa8ca381917",
                "cd4_REP4.coverage.hist.txt:md5,5888d0a421c07cd417e18b09b25b68ab",
                "cd4_REP4.coverage.stats.txt:md5,611d637d4bbdd0d96071cd1e291be5dd",
                "jurkat_REP1.coverage.hist.txt:md5,25ff1313ce15f2332b3b9ac5e440ad76",
                "jurkat_REP1.coverage.stats.txt:md5,e27a85bf418ceec8205c3ed4c9e9c0e4",
                "jurkat_REP2.coverage.hist.txt:md5,aba00d7a48866322ce46c9427023fd36",
                "jurkat_REP2.coverage.stats.txt:md5,4a5f9c89f04b4792669ca4d23bcfb66e"
            ],
            "cd4.bed:md5,74d4739278c1d8af17a5f9d3296325da",
            "jurkat.bed:md5,6d3edfc671c1bed970ca5934edc60756",
            [
                "cd4_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP1.lc_extrap.txt:md5,81f27f75f082bca4eeeaed416a484429",
                "cd4_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP2.lc_extrap.txt:md5,7c95f712b396568825145383cea0c525",
                "cd4_REP3.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP3.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP3.lc_extrap.txt:md5,e83ef4e546a1d66ec5cc5e84a9766c33",
                "cd4_REP4.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "cd4_REP4.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "cd4_REP4.lc_extrap.txt:md5,728975500782651e233de6705a044e8b",
                "jurkat_REP1.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP1.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP1.lc_extrap.txt:md5,04931737e170ed1764125f940684983e",
                "jurkat_REP2.c_curve.txt:md5,cf4743abdd355595d6ec1fb3f38e66e5",
                "jurkat_REP2.command.log:md5,d41d8cd98f00b204e9800998ecf8427e",
                "jurkat_REP2.lc_extrap.txt:md5,157f7f7532f865226da4fb460c2ff7da"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-18T19:39:47.797490045"
    }
}